docs/source/advanced_topics/0_pipeline_capacity.rst

```diff
@@ -0,0 +1,8 @@
+Pipeline Capacity
+====================
+
+Savant allows setting the pipeline capacity parameter to define how many streams a pipeline can simultaneously process. The parameter is defined with the ``max_parallel_streams`` property in the general parameters section. When the number of streams directed to the pipeline is higher than the value set by the parameter, the pipeline will terminate with the error.
+
+This parameter is set to ``64`` by default, a pretty high value for the real-time pipeline. Consider this parameter as a trigger determining the cases when you wrongly directed more streams to the pipeline than it is expected to handle on particular hardware. Keep the pipeline reasonably low for real-time pipelines to not break real-time constraints. In contrast, you may want to increase the parameter for non-real-time pipelines to ensure that the pipeline meets sharding expectations.
```

Comment:
`Keep the pipeline reasonably low for real-time pipelines to not break real-time constraints.` ->
`Keep the parameter reasonably low for real-time pipelines to not break real-time constraints.`